### PR TITLE
moar maintenance

### PR DESCRIPTION
**Pull request by TacoTheDank**

### CODE REVIEW

**Comment by tasomaniac** [comment]
Hi thanks for taking time to work in the project. Unfortunately the changes are too many. There are many unnecessary changes. Can you double check? Would be great if it is simpler and I can merge it easily

File: .gitignore [hunk 1]
```diff
@@ -1,26 +1,5 @@
-.gradle
+.gradle/
```
Comment (tasomaniac) [inline .gitignore:1]: Please do not change this file. Thanks.

File: app/build.gradle [hunk 2]
```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 apply plugin: 'com.novoda.android-command'
 apply plugin: 'com.novoda.build-properties'
+apply plugin: 'com.github.ben-manes.versions'
```
Comment (tasomaniac) [inline app/build.gradle:4]: It is enough to apply this to the main file. Not necessary here.

File: app/build.gradle [hunk 3]
```diff
@@ -30,6 +31,7 @@ android {
 
     buildTypes {
         debug {
+            zipAlignEnabled true
```
Comment (tasomaniac) [inline app/build.gradle:34]: Not necessary

File: app/proguard-rules.pro [hunk 4]
```diff
@@ -1,11 +1,11 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /Users/tasomaniac/android-sdks/tools/proguard/proguard-android.txt
+# in C:/Users/$USER/AppData/Local/Android/Sdk/tools/proguard/proguard-android.txt
```
Comment (tasomaniac) [inline app/proguard-rules.pro:3]: Please do not change this file. Thanks.

**Comment by tasomaniac** [comment]
Didn't fully review. I'm sorry to say that the PR still have many irrelevant changes.

**Comment by TacoTheDank** [comment]
you can take the changes you like and make them yourself, if you like

**Pull request closed without merging**